internal/log/logger.go

```diff
@@ -0,0 +1,62 @@
+package log
```

Comment:
Let's rename this to `internal_log` (or similar):
- It can then be in `internal_log/` directory (same level as `dat`, `kvs`, etc.)
- It conveys clearly that this is not intended to be external API, and that we're only exporting symbols for re-use internally in other packages
- Avoids clashing with `log` package of stdlib, which should be avoided (https://blog.golang.org/package-names)

Then, we can avoid renaming `log` to `stdlog` when importing which is more standard.